lib/puppet_litmus/puppet_helpers.rb

```diff
@@ -83,18 +83,20 @@ def apply_manifest(manifest, opts = {})
       command_to_run += ' --noop' if !opts[:noop].nil? && (opts[:noop] == true)
       command_to_run += ' --detailed-exitcodes' if use_detailed_exit_codes == true
 
-      command_to_run = "try { #{command_to_run}; exit $LASTEXITCODE } catch { write-error $_ ; exit 1 }" if os[:family] == 'windows'
-
       span.add_field('litmus.command_to_run', command_to_run)
```

Comment:
not capturing the actual command ran on windows, anymore.

Reply:
Moved the span inside the conditions to capture the commands